app/assets/javascripts/router.js

```diff
@@ -107,26 +101,23 @@ class Router extends BaseRouter {
 
 
   projectCreate() {
-    // Webpack `require` doesn't work with inline arrow functions
-    const callback = (ProjectCreateView, ProjectModel) => {
-      ProjectCreateView = ProjectCreateView.default;
-      ProjectModel = ProjectModel.default;
+    import(/* webpackChunkName: "admin" */ "admin/admin").then((admin) => {
```

Comment:
@daniel-wer Hahahaha. It's called `magic comments` and you can use them name dynamic chunks: https://webpack.js.org/guides/code-splitting-async/#chunk-names